### ISSUE TITLE
[Bug]: Always open Combobox doesn't announce the full control, only results

### ISSUE THREAD

**Issue by marciat** [microsoft/fluentui#28250]
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
System:
    OS: Linux 5.15 Ubuntu 22.04.1 LTS 22.04.1 LTS (Jammy Jellyfish)
    CPU: (8) x64 11th Gen Intel(R) Core(TM) i7-1185G7 @ 3.00GHz
    Memory: 7.03 GB / 15.54 GB
    Container: Yes
    Shell: 5.1.16 - /bin/bash
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://codesandbox.io/s/loving-jones-tm3tsg?file=/example.tsx

### Bug Description

Scenario description: we want to be able to render Combobox in always open state, more concretely in nested menu popover, limited to 5 results, with rest hidden behind scroll. We want to follow the pattern as in [Editable Combobox With List Autocomplete Example | APG | WAI | W3C](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-autocomplete-list/).

 

## Actual Behavior
In scenario where combobox is always open, when it is navigated to the screen reader announces and focuses on the first option of the listbox immediately. When suppressing `aria-activedescedant`, it announces the control but still focuses on the first option immediately.

 

## Expected Behavior
In scenario where combobox is always open, the screen reader  focuses on the input first and announces it, and pressing down arrow moves to the first option of the listbox.

 

As we understand the code responsible for this behavior is here:
![screenshot1](https://github.com/microsoft/fluentui/assets/37416720/3291df22-94fa-4285-87a2-106e0818b687)
![screenshot2](https://github.com/microsoft/fluentui/assets/37416720/05098bdc-2400-4acd-a520-55ba7804ace2)

Could we get a property to disable such behavior? Or maybe a change to only set active option if state changed from closed to open (not if it was open from the beginning).

### Requested priority

Blocking

### Are you willing to submit a PR to fix?

yes

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by sjwilczynski** [comment]
Interested in this as well!

**Comment by smhigley** [comment]
Could you elaborate on the use case? We were considering shipping the Listbox as a standalone control (the bit inside the popup for Combobox, independent of the input + popup behavior). If you want it to always be open, that sounds like potentially a better fit.

That said, putting it within a menu sounds like maybe Combobox/Listbox is not the right control at all, since combobox/listbox/option semantics won't be valid within a menu popup.

**Comment by sjwilczynski** [comment]
We discussed it and realized combobox was not the right control for such situation, as we will be rendering from within a menu. Thanks for you feedback!
